package.json

```diff
@@ -37,7 +37,7 @@
     "@ethereumjs/tx": "^3.2.1",
     "ethereumjs-util": "^7.0.9",
     "hdkey": "0.8.0",
```

Comment:
Side-note, I'm not touching this line in this PR, but should `hdkey` be updated?

According to https://raw.githubusercontent.com/MetaMask/metamask-extension/develop/yarn.lock, only eth-trezor-keyring and @metamask/eth-ledger-bridge-keyring are using this old version, so it might be worth updating.